AI #1663: Billed Cost and Effective Cost definitions update (#1678)
FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec: specification/datasets/cost_and_usage/columns/billedcost.md

# Billed Cost

> Billed Cost represents the cost of a [*charge*](#glossary:charge) as invoiced by the [invoice issuer](#datasets.costandusage.invoiceissuername) in a given [*billing period*](#glossary:billing-period). Billed Cost differs from [Effective Cost](#datasets.costandusage.effectivecost) when *charges* are invoiced separately from usage.

Reviewer: ```suggestion
Billed Cost represents the cost of a [*charge*](#glossary:charge) as invoiced by the [invoice issuer](#datasets.costandusage.invoiceissuername) in a given [*billing period*](#glossary:billing-period). Billed Cost differs from [Effective Cost](#datasets.costandusage.effectivecost) when *charges* (both pre-paid and post-paid) are invoiced separately from usage.
```

Reviewer 2: Looks good to me!

> Billed Cost represents the cost of a [*charge*](#glossary:charge) invoiced (or to be invoiced) by the [invoice issuer](##datasets.costandusage.invoiceissuername) in a given [*billing period*](#glossary:billing-period). Billed Cost differs from [Effective Cost](#datasets.costandusage.effectivecost) when pre-paid or post-paid charges are invoiced separately from usage.

Reviewer 2: Do we truly need the `pre-paid or post-paid` chunk?  I found myself reading that sentence over and over, struggling to grok it, and removing it really seems to help my understanding.  In other words: if all charges are pre-paid or post-paid, then we can just say "charges". If pre-paid and post-paid is not all charges, then what a term that describes what is left over?

Is it possible for us to simplify `invoiced (or to be invoiced)` to just `invoiced`, if we add an entry in the glossary for "invoice process" (or similar)?  In that entry, we could describe the invoicing process at a high level (i.e., charges result, eventually, in invoices), which would allow us to avoid needing to specify the difference between past-tense invoiced and future-tense to-be-invoiced.  And, any time we use the term "invoiced", we could link to that glossary entry to help people understand the process.

(To be clear: this PR could assume the creation of that glossary entry, per the work I am doing in TF-1 against FR #1500.  This PR would not need to create it.)

```suggestion
Billed Cost represents the cost of a [*charge*](#glossary:charge) as [*invoiced*](#glossary:invoice-process) by an [invoice issuer](##datasets.costandusage.invoiceissuername) in a given [*billing period*](#glossary:billing-period). Billed Cost differs from [Effective Cost](#datasets.costandusage.effectivecost) when *charges* are *invoiced* separately from usage.
```

Reviewer: These have been discussed in TF as well. In my view, it doesn't hurt to be explicit (painfully so) to avoid assumptions people might make that something applies only when prepayments are involved (which is often what people think of when they think of EffectiveCost), but it's true that some post-paid scenarios can be relevant as well. Not a strong view either way. If we see the value, then let's keep it; removing is easy.

Reviewer: Final edits based on 1/23 TF3. Group decided to leave invoice-process glossary out for now till it exists. 'an' Invoice Issuer changed back to 'the' Invoice Issuer
```suggestion
Billed Cost represents the cost of a [*charge*](#glossary:charge) as invoiced by the [invoice issuer](#datasets.costandusage.invoiceissuername) in a given [*billing period*](#glossary:billing-period). Billed Cost differs from [Effective Cost](#datasets.costandusage.effectivecost) when *charges* are invoiced separately from usage.
```

> For all *charges*, Billed Cost reflects all applicable pricing adjustments, such as reduced pricing from negotiated or [*commitment discounts*](#glossary:commitment-discount). For purchase *charges*, Billed Cost includes any amounts invoiced in the given *billing period*. For usage *charges*, Billed Cost excludes any amounts that are or will be covered by related purchase *charges* (e.g., *commitment* purchases, pre-paid contracts, or marketplace purchases), regardless of when those related charges are invoiced.

Reviewer 2: Attempting to harmonize with how the same section of Effective Cost is expressed, including similar revisions from my suggestion [here](https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/pull/1678/files#r2713665066).

I'm also suggesting "portion" here instead of "amount".  In my opinion, that's clearer that we may be talking about 0%, 100%, or some percentage in-between, depending on the circumstance.  And that aligns with the use of "portion" from Effective Cost.

```suggestion
For all *charges*, Billed Cost reflects all applicable pricing adjustments (e.g., reduced pricing from [*negotiated discounts*](#glossary:negotiated-discount) or [*commitment discounts*](#glossary:commitment-discount)). For purchase *charges*, Billed Cost includes any portion invoiced in the given *billing period*. For usage *charges*, Billed Cost excludes any portion of related purchase *charges* (e.g., usage covered by *commitments*, pre-payments, or marketplace purchases), regardless of when those related charges are invoiced.
```

Reviewer: Incorporating Shawn's updates and moving the 'covered' word outside of the e.g.)

```suggestion
For all *charges*, Billed Cost reflects all applicable pricing adjustments (e.g., reduced pricing from [*negotiated discounts*](#glossary:negotiated-discount) or [*commitment discounts*](#glossary:commitment-discount)). For purchase *charges*, Billed Cost includes any portion invoiced in the given *billing period*. For usage *charges*, Billed Cost excludes any portion covered by related purchase *charges* (e.g., *commitments*, pre-payments, or marketplace purchases), regardless of when those related charges are invoiced.
```

Reviewer 2: Per Jan 22 Members: we may need to add a "covered" glossary entry that we can link here.

# Billed Cost

> Billed Cost represents the amount to be invoiced for a [*charge*](#glossary:charge) by the [invoice issuer](#invoiceissuername) for a given [*billing period*](#glossary:billing-period). It is inclusive of the effects of all applicable pricing adjustments, such as reduced pricing from [*negotiated*](#glossary:negotiated-discount) or [*commitment*](#glossary:commitment-discount) discounts. It is exclusive of any prepaid portion of the charge. This cost is denominated in the [Billing Currency](#billingcurrency). Billed Cost is commonly used for activities such as invoice reconciliation, budgeting, and forecasting and cost allocation based on invoiced amounts.

Author: ### OPTION 1' 

*formulated during the Jan 09, 2026 TF-3 meeting*

```suggestion
Billed Cost represents the amount to be invoiced for a [*charge*](#glossary:charge) by the [invoice issuer](#invoiceissuername) for a given [*billing period*](#glossary:billing-period). It is inclusive of the effects of all applicable pricing adjustments, such as reduced pricing from [*negotiated*](#glossary:negotiated-discount) or [*commitment*](#glossary:commitment-discount) discounts. It is exclusive of any prepaid portion of the charge. This cost is denominated in the [Billing Currency](#billingcurrency). Billed Cost is commonly used for activities such as invoice reconciliation, budgeting, and forecasting and cost allocation based on invoiced amounts.
```

Author: resolving in favor of [this comment](https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/pull/1678#discussion_r2699431892)

> Billed Cost represents the amount to be invoiced for a [*charge*](#glossary:charge) by the [invoice issuer](#invoiceissuername) for a given [*billing period*](#glossary:billing-period). It is inclusive of the effects of all applicable pricing adjustments, such as reduced pricing from [*negotiated*](#glossary:negotiated-discount) or [*commitment*](#glossary:commitment-discount) discounts. It is exclusive of any prepaid portion of the charge. This cost is denominated in the [Billing Currency](#datasets.costandusage.billingcurrency). Billed Cost is commonly used for activities such as invoice reconciliation, budgeting, and forecasting and cost allocation based on invoiced amounts.

Author: ### OPTION 1' 

*formulated during the Jan 09, 2026 TF-3 meeting*

```suggestion
Billed Cost represents the amount to be invoiced for a [*charge*](#glossary:charge) by the [invoice issuer](#invoiceissuername) for a given [*billing period*](#glossary:billing-period). It is inclusive of the effects of all applicable pricing adjustments, such as reduced pricing from [*negotiated*](#glossary:negotiated-discount) or [*commitment*](#glossary:commitment-discount) discounts. It is exclusive of any prepaid portion of the charge. This cost is denominated in the [Billing Currency](#datasets.costandusage.billingcurrency). Billed Cost is commonly used for activities such as invoice reconciliation, budgeting, and forecasting and cost allocation based on invoiced amounts.
```

> Billed Cost represents the amount to be invoiced for a [*charge*](#glossary:charge) by the [invoice issuer](#invoiceissuername) for the specified [*billing period*](#glossary:billing-period). This cost metric enables cash-basis accounting by aligning [*charge-level*](#glossary:charge) costs with the actual amounts invoiced or to be invoiced at the end of the *billing period*. Billed Cost is inclusive of the effects of all applicable pricing adjustments, such as reduced unit prices resulting from [*negotiated*](#glossary:negotiated-discount) or [*commitment discounts*](#glossary:commitment-discount). This cost is denominated in the [Billing Currency](#billingcurrency). Billed Cost is commonly used to support FinOps capabilities, such as invoice reconciliation, cash-flow-based forecasting, budgeting, and cost allocation.

Reviewer 3: I do not like the inclusion of "cash-based accounting" since I feel like organizations are cash based or accrual based and "cash-based accounting" is an extremely broad thing that may include other aspects of revenue recognition or other accounting-y things that I, as a non-accountant, would not mean to be implying. 

I also don't like Accrual based accounting below for the same reason, but i'll comment there. 

I would suggest just starting the second sentence with "Billed Cost is the..." then pick up after aligning. 

Please remove the word "capabilities" from the last sentence, since those things are not FinOps Capabilities and this might create confusion. We could substitute the word "activities" 

The rest of the definition seems good to me, and the inclusion of the "cash-flow-based forecasting" in the last sentence gets to the heart of I think what was intended by including "cash-based accounting" but is much more specific and descriptive and less dangerous. We could also say in that last part that Billed Cost is what is used by Accounts Payable to pay the invoice issuer.

Author: Alternative, without using the term cash-based accounting, while retaining an emphasis on invoiced amount in a given billing period.

```suggestion
Billed Cost represents the amount to be invoiced for a [*charge*](#glossary:charge) by the [invoice issuer](#invoiceissuername) for a given [*billing period*](#glossary:billing-period). It is inclusive of the effects of all applicable pricing adjustments, such as reduced unit prices resulting from [*negotiated*](#glossary:negotiated-discount) or [*commitment discounts*](#glossary:commitment-discount). This cost is denominated in the [Billing Currency](#billingcurrency). Billed Cost is commonly used for activities such as invoice reconciliation, budgeting, and forecasting and cost allocation based on invoiced amounts.
```

Author: Resolved per consensus reached during the TF-3 meeting on Jan 09, 2026. 
@robmartin33 verbally confirmed that the update agreed during that meeting and committed on Jan 16, 2026 addresses all concerns raised in this comment.

## Description

> Amount to be invoiced for a *charge* by the invoice issuer for the specified *billing period*.

Author: ```suggestion
Amount to be invoiced for a *charge* by the invoice issuer for a given *billing period*.
```

Author: Resolved per consensus reached during the TF-3 meeting on Jan 09, 2026.
Update agreed during that meeting and committed on Jan 16, 2026.

## Description

> Cost of a *charge* invoiced (or to be invoiced) by the *invoice issuer* in a given *billing period*.

Reviewer 2: Revision to bring it in line with the latest longer definition above.

```suggestion
Cost of a *charge* as invoiced by the *invoice issuer* in a given *billing period*.
```
